File: src/main/Python/task_1/accounting.py
from employee import Employee


class Accounting:
    employee: Employee

    def __init__(self, employee):
        self.employee = employee

    def calculate_net_salary(self):
        tax = int(self.employee.base_salary * 0.25)  # рассчитать налог другим способом
        return self.employee.base_salary - tax

File: src/main/java/task_1/Accounting.java
package task_1;

public class Accounting {
    private Employee employee;

    public Accounting(Employee employee) {
        this.employee = employee;
    }

    public int calculateNetSalary() {
        int tax = (int) (employee.getBaseSalary() * 0.25);
        return employee.getBaseSalary() - tax;
    }
}
